run.sh

```diff
@@ -0,0 +1,57 @@
+# Argument Parsing
+echo "Positional Parameters"
```

Comment:
Please change this so that the usage information is only printing when the users did not use the script correctly.